perun-openapi/openapi.yml

```
@@ -5454,6 +5468,64 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/usersManager/createAlternativePassword:
+    post:
+      tags:
+        - UsersManager
+      operationId: createAlternativePassword
+      summary: Creates alternative password in external system.
+      parameters:
+        - $ref: '#/components/parameters/userId'
+        - name: description
+          schema: {
+            type: string
+          }
+          in: query
+          required: true
+        - name: loginNamespace
+          schema: {
+            type: string
+          }
+          in: query
+          required: true
+        - name: password
+          schema: {
+            type: string
+          }
+          in: query
+          required: true
```

```suggestion
        - name: description, schema: { type: string }, in: query, required: true
        - name: loginNamespace, schema: { type: string }, in: query, required: true
        - name: password, schema: { type: string }, in: query, required: true
```
Please use the shorter YAML syntax for parameters, it is easier to read.